src/Operation/MapReduce.php

```diff
@@ -17,6 +17,7 @@
 
 namespace MongoDB\Operation;
 
+use MongoDB\BSON\Javascript;
```

Comment:
`finalize` should actually check for `JavascriptInterface`. I've opened [PHPLIB-313](https://jira.mongodb.org/browse/PHPLIB-313) to track that, and we'll fix this in 1.3.0.